pkg/skaffold/jib/jib_init_test.go

```diff
@@ -187,23 +187,19 @@ func TestCreateArtifact(t *testing.T) {
 			config:        Jib{BuilderName: "Jib Gradle Plugin", FilePath: filepath.Join("path", "to", "build.gradle"), Image: "image", Project: "project"},
 			manifestImage: "different-image",
 			expectedArtifact: latest.Artifact{
-				ImageName: "different-image",
-				Workspace: filepath.Join("path", "to"),
-				ArtifactType: latest.ArtifactType{
-					JibArtifact: &latest.JibArtifact{Project: "project", Flags: []string{"-Dimage=different-image"}, Type: string(JibGradle)},
-				},
+				ImageName:    "different-image",
+				Workspace:    filepath.Join("path", "to"),
+				ArtifactType: latest.ArtifactType{JibArtifact: &latest.JibArtifact{Project: "project", Type: string(JibGradle)}},
 			},
 		},
 		{
 			description:   "jib gradle without image and project",
 			config:        Jib{BuilderName: "Jib Gradle Plugin", FilePath: filepath.Join("path", "to", "build.gradle")},
 			manifestImage: "different-image",
 			expectedArtifact: latest.Artifact{
-				ImageName: "different-image",
-				Workspace: filepath.Join("path", "to"),
-				ArtifactType: latest.ArtifactType{
-					JibArtifact: &latest.JibArtifact{Flags: []string{"-Dimage=different-image"}, Type: string(JibGradle)},
-				},
+				ImageName:    "different-image",
+				Workspace:    filepath.Join("path", "to"),
+				ArtifactType: latest.ArtifactType{JibArtifact: &latest.JibArtifact{Type: string(JibGradle)}},
```

Comment:
About https://github.com/GoogleContainerTools/skaffold/issues/3190#issue-518528937, is it easy to get rid of `type: gradle` or it is safe to put them?

Reply:
@briandealwis would know better than I.

Reply:
It should be safe to remove.